docker/package/scripts/tezos-node-start

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/bash
+
+# SPDX-FileCopyrightText: 2020 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+set -euo pipefail
+
+: "${BIN_DIR:="/usr/bin"}"
+: "${DATA_DIR:="/var/lib/tezos"}"
+
+node="$BIN_DIR/tezos-node"
+node_dir="$DATA_DIR/node"
+node_data_dir="$node_dir/data"
+
+launch_node() {
+    mkdir -p "$node_data_dir"
+    if [ ! -f "$node_data_dir/config.json" ]; then
+        echo "Configuring the node..."
+        "$node" config init \
+                --data-dir "$node_data_dir" \
+                --rpc-addr ":$NODE_RPC_PORT" \
+                "$@"
+    else
+        echo "Updating the node configuration..."
+        "$node" config update \
+                --data-dir "$node_data_dir" \
+                --rpc-addr ":$NODE_RPC_PORT" \
+                "$@"
+    fi
+    # Generate a new identity if not present
+
+    if [ ! -f "$node_data_dir/identity.json" ]; then
+        echo "Generating a new node identity..."
+        "$node" identity generate "${IDENTITY_POW:-26}". \
+                --data-dir "$node_data_dir"
+    fi
+
+    # Launching the node
+
+    exec "$node" run --data-dir "$node_data_dir"
+
+}
+
+launch_node --network "$NETWORK" "$@"
```

Comment:
Not much of a point in defining a function just to call it once at the bottom.

Reply:
updated